.config/sway/config.d/50-openSUSE.conf

```diff
@@ -37,8 +37,8 @@ bindsym --to-code {
 # Media keys
 bindsym XF86AudioMicMute exec pactl set-source-mute @DEFAULT_SOURCE@ toggle
 
-bindsym XF86MonBrightnessDown exec brightnessctl -q set 5%- && ( echo $((`brightnessctl get` * 100 / `brightnessctl m`)) > $SWAYSOCK.wob )
-bindsym XF86MonBrightnessUp exec brightnessctl -q set +5% && ( echo $((`brightnessctl get` * 100 / `brightnessctl m`)) > $SWAYSOCK.wob )
+bindsym XF86MonBrightnessDown exec brightnessctl -q -e set 5%- && ( perl -e "use Math::Complex; printf \"%.0f\n\", 100 * ((`brightnessctl get` / `brightnessctl m`)**0.25)" > $SWAYSOCK.wob )
```

Comment:
Pulling in yet another programming language (Perl) is not a good idea. Look at https://github.com/openSUSE/openSUSEway/issues/126 as well.